src/components/pageTemplates/NodeEventPage.tsx

```diff
@@ -172,27 +171,21 @@ function NodeEventPage({ node, ...props }: NodeEventPageProps): JSX.Element {
                 {field_in_language.length > 0 && (
                   <SideContent
                     header={t('event.languages')}
-                    content={event_languages.toString().replace(',', ', ')}
                     icon={<IconGlobe />}
-                  />
+                  >
+                    {event_languages.toString().replace(',', ', ')}
```

Comment:
This results in "Englanti, Suomi,Ruotsi", so only the first comma gets a space after it.